modules/ROOT/examples/java/src/com/couchbase/code_snippets/Examples.java

```diff
@@ -23,6 +23,12 @@
 /*      Credentials declared this way purely for expediency in this demo - use OAUTH in production code */
 private static final String DB_USER = "sync_gateway";
 private static final String DB_PASS = "password"; // <3>
+<<<<<<< Updated upstream
```

Comment:
You need to edit down these two git diffs

Reply:
Sorry, @RichardSmedley; done now. These don't seem to show up  in Atom; done now